src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -122,68 +117,140 @@ public static void tearDown() {
+            JsonUtils.toJson(otpUser),
+            headers,
+            HttpUtils.REQUEST_METHOD.POST
         );
-        OtpDispatcherResponse otpDispatcherResponse = new OtpDispatcherResponse(mockResponse, DEFAULT_PLAN_URI);
+
+        assertEquals(HttpStatus.OK_200, createUserResponse.statusCode());
+
+        // Attempt to create a monitored trip for an Otp user authenticating as an Otp user. This will fail because the
+        // user was created by an Api user and therefore does not have a Auth0 account.
+        OtpUser otpUserResponse = JsonUtils.getPOJOFromJSON(createUserResponse.body(), OtpUser.class);
 
         // Create a monitored trip for the Otp user (API users are prevented from doing this).
         MonitoredTrip monitoredTrip = new MonitoredTrip(TestUtils.sendSamplePlanRequest());
         monitoredTrip.userId = otpUser.id;
-        HttpResponse<String> createTripResponse = mockAuthenticatedPost("api/secure/monitoredtrip",
+        HttpResponse<String> createTripResponseAsOtpUser = mockAuthenticatedRequest("api/secure/monitoredtrip",
+            JsonUtils.toJson(monitoredTrip),
             otpUserResponse,
-            JsonUtils.toJson(monitoredTrip)
+            HttpUtils.REQUEST_METHOD.POST,
+            true
         );
-        assertEquals(HttpStatus.OK_200, createTripResponse.statusCode());
-        MonitoredTrip monitoredTripResponse = JsonUtils.getPOJOFromJSON(createTripResponse.body(), MonitoredTrip.class);
+        assertEquals(HttpStatus.UNAUTHORIZED_401, createTripResponseAsOtpUser.statusCode());
 
-        // Plan trip with OTP proxy. Mock plan response will be returned
-        String otpQuery = OTP_PROXY_ENDPOINT + OTP_PLAN_ENDPOINT + "?fromPlace=28.45119,-81.36818&toPlace=28.54834,-81.37745";
-        HttpResponse<String> planTripResponse = mockAuthenticatedRequest(otpQuery,
-            otpUserResponse,
+        // Create a monitored trip for an Otp user authenticating as an Api user. An Api user can create a monitored
+        // trip for an Otp user they created.
+        HttpResponse<String> createTripResponseAsApiUser = authenticatedRequest("api/secure/monitoredtrip",
+            JsonUtils.toJson(monitoredTrip),
+            headers,
+            HttpUtils.REQUEST_METHOD.POST
+        );
+        assertEquals(HttpStatus.OK_200, createTripResponseAsApiUser.statusCode());
+        MonitoredTrip monitoredTripResponse = JsonUtils.getPOJOFromJSON(createTripResponseAsApiUser.body(), MonitoredTrip.class);
+
+        // Request all monitored trip for an Otp user authenticating as an Api user.
+        HttpResponse<String> getAllMonitoredTripsForOtpUser = authenticatedGet(String.format("api/secure/monitoredtrip?userId=%s",
+            otpUserResponse.id),
+            headers
+        );
+        assertEquals(HttpStatus.OK_200, getAllMonitoredTripsForOtpUser.statusCode());
+
+        // Request all monitored trip for an Otp user authenticating as an Api user. Without defining the user id.
```

Comment:
Typo "all monitored trips". Also add in comment the expected outcome as you did for the other tests.

Reply:
Updated.